app/models/acted_in.rb

```diff
@@ -0,0 +1,8 @@
+class ActedIn
```

Comment:
One thing that I noticed later is that if you, for example, call `movie.people.each_with_rel` (or `each_rel`) you don't get the relationships wrapped in the `ActiveRel` module.  Apparently you need to make sure the `ActedIn` model has been referenced and then the gem will wrap the relationships.  I haven't figured out the best way to do this yet ;)

Also if you want some more detail about `ActiveRel` and other bits of the gem I released a set of screencasts recently:

https://www.youtube.com/playlist?list=PL5klM3mD6alLUhNTPTbj5a3GBjU7oZN0t